src/editor/sidemenu/UserListModel.cpp

```diff
@@ -40,9 +40,9 @@ QVariant UserListModel::data(const QModelIndex &index, int role) const {
         return user->first;
       case UserListModel::Column::Ping:
         if (user->second.last_ping.has_value()) {
-          int ping = user->second.last_ping.value();
+          int ping = static_cast<int>(user->second.last_ping.value());
           if (ping > 1000) return QString("%1s").arg(ping / 1000);
-          return QString("%1ms").arg(ping);
+          return ping + tr("ms");
```

Comment:
Does it even make sense to have these translated? I'd assume this specific part of the SI is adopted well enough that everyone either uses or at least understands `s` & `ms`.

If keeping it: Does it make sense to include the number in the string? I don't know if/how simple appending or the translation would interact with RTL languages.

```suggestion
          return tr("%1ms").arg(ping);
```

Reply:
Yeah slight favour not to have it since s / ms seems p standard to me